client/record/cli/query.go

```diff
@@ -0,0 +1,87 @@
+package cli
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/cosmos/cosmos-sdk/wire"
+	"github.com/irisnet/irishub/client"
+	"github.com/irisnet/irishub/client/context"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	ctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+func GetCmdQureyHash(cdc *wire.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "tx [hash]",
+		Short: "Matches this txhash over all committed blocks",
```

Comment:
Could we improve this msg to be more readable?

Reply:
This msg is already updated in latest commit.